clean_project/project/search_engine.py

```diff
@@ -21,11 +23,50 @@ def search(self, search_word):
         sslmode = load_config_file(self.connect_path).get('sslmode')
         connection = psycopg2.connect(host=host, port=port, sslmode=sslmode,
                                       database=database, user=user, password=password)
+        words = tuple(search_word.split(" "))
```

Comment:
Because in the Select query is need to build a tuple for  IN clause.

example: select * from index where word in ("word1", "word2", "word3");